hypha/apply/determinations/templates/determinations/base_determination_form.html

```diff
@@ -54,22 +62,24 @@ <h2>{{ value }}</h2>
                     {{ message }}
                 </div>
             {% endfor %}
-            <aside id="submission-flyout"
-                   style="margin: 1rem"
-                   x-bind:style="submissionVisible ? 'flex-basis: 50%; margin: 1rem' : 'flex-basis: 2rem; margin: 0'">
-                <button class="button button--primary"
-                        hx-get="{% url 'funds:submissions:partial-answers' submission.id %}"
-                        hx-trigger="click once"
```

Comment:
The button seems to make a request on every click (whether on show or hide). I wonder if the removal of `hx-trigger="click once"` is related. Is this desired behavior? I would expect the request to happen exactly once for the page. Barring that, I would expect it to happen only when the submission gets shown, right?

Reply:
Good catch, will test this. I copied most of this from the mobil nav menu.

Reply:
I think it works better now, removed "intersect". Appreciate some testing to confirm that is only loads once.

Reply:
Yes, it worked! I made the same change on the other form and it seems to work there too (removing `intersect` and the comma).